### ISSUE TITLE
Hide "Påmelding åpner" after event actually opened

### ISSUE THREAD

**Issue by sklirg** [dotkom/onlineweb4#1058]
![screenshot 2015-02-11 14 53 50](https://cloud.githubusercontent.com/assets/5422571/6148075/72154f9c-b1fe-11e4-9746-53c528b05f6f.png)

It looks weird with the one line on a new line after the three other info-boxes.

There are 4 "boxes" of information, each spanning 3 + 3 + 3 + 5 columns. This is more than 12 columns, causing the last one to appear on a new line.

We should look into what information is relevant here.

When the event opened is not so relevant after it has opened. Maybe we could:
- Show when the attending closes (or when it closed, if it's closed)
- Hide information about when it opened

We could also remove information about attending status if the waiting list is populated, since this implies that the event is full. This leads to the problem what to show if event is full, either show e.g. `25/25 attending` OR `Waiting list: 0`.

![screenshot 2015-02-11 14 57 04](https://cloud.githubusercontent.com/assets/5422571/6148076/73c97ec6-b1fe-11e4-91d9-63a76c3bcf3a.png)

This event is not open, therefore has no waiting list, and all information is on one line.

**TL;DR**: Make information about events in archive-view be on one line
